### DISCUSSION TITLE
Delete comments and receive notifications as an admin

### DISCUSSION THREAD

**Discussion by criticalcake** [umputun/remark42#926] [Q&A] [answered]
Hi there,

as a site admin I would like to receive notifications each time someone creates a comment on my site.
Also, I would like to be able to delete comments, which seem to be inappropriate (e.g. SPAM, etc.).
At the moment, I don't use telegram and I don't have any kind of mail server installed on my VPS.

What is the recommended way to do this?

Sorry, if this is a silly question, but I checked the documentation several times and unfortunately I don't get it (as I'm new to linux and web development). 

What are valid values for ADMIN_SHARED_ID? My github account name?
Do I need a mail server? Should I use RSS feeds instead?

Many thanks in advance!

**Comment by umputun** [comment, marked as answer]
> as a site admin I would like to receive notifications each time someone creates a comment on my site.

I guess you meant email notification? This functionality already available, see `NOTIFY_EMAIL_ADMIN` (notify admin on new comments via ADMIN_SHARED_EMAIL)

> Also, I would like to be able to delete comments

Admins can delete comments

![ncsev-202103-26114003-s3yzd](https://user-images.githubusercontent.com/535880/112664516-0b0a6a00-8e28-11eb-9451-19b8241e6e6f.png)

> What are valid values for ADMIN_SHARED_ID? My github account name?

This is the id you can see after the login

<img width="413" alt="image" src="https://user-images.githubusercontent.com/535880/112664673-3725eb00-8e28-11eb-960a-a1eadcd7930b.png">

> Do I need a mail server? Should I use RSS feeds instead?

Up to you. If you want to get email notification you need to set up some email service. See [documentation](https://remark42.com/docs/1.6/email/) covering integration with popular services.

**Comment by criticalcake** [comment]
Hey @umputun,

this is exactly what I was looking for.
Thank you so much for your kind help 🙏 and the Remark42 commenting engine 🎆. Great work!  👍
